source/installing-with-npm/index.html.md.erb

```diff
@@ -66,7 +66,7 @@ The accordion will use a generic font until you get the font and images working,
 There are also different ways you can [import GOV.UK Frontend's CSS](../import-css/), including into your project's main Sass file:
 
     ```scss
-    @import "node_modules/govuk-frontend/dist/govuk/all";
+    @import "node_modules/govuk-frontend/dist/govuk/index";
     ```
```

Comment:
**suggestion** For consistency with the other examples about `index` files 😊 

```suggestion
    @import "node_modules/govuk-frontend/dist/govuk/index";
    ```

You do not need `/index` at the end of your import if you’re using Dart Sass, LibSass 3.6.0 or higher, or Ruby Sass 3.6.0 or higher.
```

Reply:
I thought about doing this but wondered if the link to the sass importing docs above carried that context. It can't hurt to put it here though so I'll commit this.